### ISSUE TITLE
Unable to find secret env from yatai-system namespace

### ISSUE THREAD

**Issue by kimkihoon0515** [bentoml/Yatai#436]
Here's how it's written in the readme.
```
YATAI_INITIALIZATION_TOKEN=$(kubectl get secret env --namespace yatai-system -o jsonpath="{.data.YATAI_INITIALIZATION_TOKEN}" | base64 --decode)
echo "Open in browser: http://127.0.0.1:8080/setup?token=$YATAI_INITIALIZATION_TOKEN"
```
But I can't get a URL from it because there's no secret named env in yatai-system namespace.
<img width="243" alt="image" src="https://user-images.githubusercontent.com/63439911/212534636-3a2f00aa-42f0-429f-9899-ecd94b95c983.png">
By replacing env with yatai-env I got the right result.
Can u please fix secret env to secret yatai-env to avoid confusion??

**Comment by KimSoungRyoul** [comment]
it seems like already fixed issue in `v1.0.0-d14 b`
https://github.com/bentoml/Yatai/issues/382

try to `helm repo update bentoml`

**Comment by yetone** [comment]
Thanks for your report! I have already fixed it now: https://github.com/bentoml/Yatai/commit/bba90ec44290d33d49aa861953538edf32265014
